client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1127,6 +1127,15 @@ Undocumented.prototype.readA8C = function( query, fn ) {
 	return this.wpcom.req.get( '/read/a8c', query, fn );
 };
 
+Undocumented.prototype.readConversations = function( query, fn ) {
```

Comment:
Imagine a file in the data-layer called `wpcom-http-actions` ands its just full of http action creators.